web/src/user_profile.js

```diff
@@ -252,10 +273,25 @@ export function show_user_profile(user, default_tab_key = "profile-tab") {
                 case "user-profile-streams-tab":
                     render_user_stream_list(user_streams, user);
                     break;
+                case "manage-profile-tab":
+                    render_manage_profile_content(user);
+                    break;
             }
         },
     };
 
+    if (can_manage_profile) {
+        let manage_profile_tab_label = "Manage user";
+        if (user.is_bot) {
+            manage_profile_tab_label = "Manage bot";
+        }
+        const manage_profile_tab = {
+            label: $t({defaultMessage: "{manage_profile_tab_label}"}, {manage_profile_tab_label}),
```

Comment:
Let's double-check that this is okay for doing translations; most other uses of `defaultMessage` with interpolated strings have just the part that differs ("user", "bot" in this case).

Reply:
Thanks for noticing, fixed it